spec/system/admin_employee_manage_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe 'It enables to manage employee accounts', type: :system do
+  before do
+    admin = create(:admin)
+    create(:employee)
+    login_as(admin, scope: :admin)
+    driven_by(:selenium_chrome_headless)
+  end
+
+  it 'checks if employees are listed?' do
+    visit '/admins/employees'
+    expect(page).to have_link 'Update profile'
```

Comment:
it would be better to check if the page has employee's email than to check if there is a link